package/crossplane.yaml

```diff
@@ -3,6 +3,32 @@ kind: Provider
+    friendly-name.meta.crossplane.io: Provider Helm
+    friendly-group-name.meta.crossplane.io/helm.crossplane.io: Helm Charts
+    friendly-kind-name.meta.crossplane.io/release.helm.crossplane.io: Helm Release
```

Comment:
Does it make sense for `helm.crossplane.io` group to have a friendly name of `Helm Charts`? Maybe so, but it does feel a little strange for `Helm Release` to be a kind in the `Helm Chart` group.

Reply:
Okay, I removed "Charts".